src/pseudopeople/entities.py

```diff
@@ -44,6 +47,21 @@ class __NoiseTypes(NamedTuple):
     PHONETIC: ColumnNoiseType = ColumnNoiseType(
         "phonetic", noise_functions.generate_phonetic_errors
     )
+    MISSING_DATA: ColumnNoiseType = ColumnNoiseType(
```

Comment:
why did this get added in this PR?

Reply:
Because I need to have agreement on strings that are in the YAML.